packages/rpcs/eth_signTransaction/index.js

```diff
@@ -102,7 +102,10 @@ export const main = async args => {
     )
 
   const legacyTx = !newTx.type || newTx.type === '0x0'
-  if (!legacyTx) throw InvalidParams(`Fluent don't support EIP-1559 yet`)
+  if (!legacyTx)
+    throw InvalidParams(
+      `Invalid transaction params: params specify an EIP-1559 transaction but the current network does not support EIP-1559`,
```

Comment:
Fluent does not support EIP-1559 and the network may support EIP-1559, should change the error message